__tests__/app.tests.js

```diff
@@ -106,4 +106,46 @@ describe('GET /api/articles/:article_id', () => {
       expect(response.body.msg).toBe('Invalid data type')
     });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('Status 200: responds with a specific article, given the article id and the corresponding comments sorted newest to oldest', ()=> {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((response) => {
+      const articleOneComments = response.body.comments;
+      console.log(articleOneComments)
+      expect(articleOneComments).toHaveLength(11)
+      articleOneComments.forEach((comment) => {
+        expect(comment).toMatchObject({
+          article_id: expect.any(Number),
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String)
+
+      })
+    });
+    expect(articleOneComments).toBeSortedBy("created_at", {descending: true, coerce: true, })
+  });
```

Comment:
Another test where we expect a 200 when the ID is valid but has no comments -> responds with an empty array of comments